web-ui/modules/i18n/locales/zh.mjs

```
@@ -0,0 +1,1094 @@
+    'common.refresh': '刷新',
+    'common.refreshing': '刷新中...',
+    'common.loading': '加载中...',
+    'common.saving': '保存中...',
+    'common.sending': '发送中...',
+    'common.scanning': '扫描中...',
+    'common.export': '导出',
+    'common.import': '导入',
+    'common.apply': '应用',
+    'common.applying': '应用中...',
+    'common.confirming': '确认中...',
+    'common.writeToEditor': '写入编辑器',
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)